modules/virt-vm-migration-tuning.adoc

```diff
@@ -0,0 +1,23 @@
+
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-live-migration.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-vm-migration-tuning_{context}"]
+= VM migration tuning
+
+Live migration requires the use of a shared storage solution that provides `ReadWriteMany` (RWX) access mode. VM disks must be backed by volumes defined with RWX.
+
+You can adjust your cluster-wide live migration limit settings based on the type of workload and migration scenario. This enables you to control how many VMs migrate at the same time, the network bandwidth you want to use for each migration, and how long {VirtProductName} attempts to complete the migration before canceling the process. Configure these settings in the `HyperConverged` custom resource (CR).
+
+If you are migrating multiple VMs per node at the same time, set a `bandwidthPerMigration` limit to prevent a large or busy VM from using a large portion of the node’s network bandwidth. By default, the `bandwidthPerMigration` value is `0`, which means unlimited.
+
+A large VM running a heavy workload (for example, database processing), with higher memory dirty rates, requires a higher bandwidth to complete the migration.
+
+[NOTE]
+====
+Post copy mode triggers if the initial pre-copy phase does not complete within the configured timeout parameter. During post copy, the VM CPUs pause on the source host while transfering the minimum required memory pages. Then the VM CPUs activate on the destination host, and the remaining memory pages transfer into the destination node at runtime. This can impact performance during the transfer.
```

Comment:
```suggestion
Post copy mode, when enabled, triggers if the initial pre-copy phase does not complete within the configured timeout parameter. During post copy, the VM CPUs pause on the source host while transfering the minimum required memory pages. Then the VM CPUs activate on the destination host, and the remaining memory pages transfer into the destination node at runtime. This can impact performance during the transfer.
```